tests/codegen/call_nsecs.cpp

```diff
@@ -0,0 +1,14 @@
+#include "common.h"
+
+namespace bpftrace {
+namespace test {
+namespace codegen {
+
+TEST(codegen, call_nsecs)
```

Comment:
nit
```suggestion
TEST(codegen, call_nsecs_tai)
```
same for the file name